recipes/pypydispatcher/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "PyPyDispatcher" %}
+{% set version = "2.1.2" %}
+{% set sha256 = "b6bec5dfcff9d2535bca2b23c80eae367b1ac250a645106948d315fcfa9130f2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - setuptools
+  run:
+    - python >=3
+
+test:
+  imports:
+    - pydispatch
+
+about:
+  home: https://github.com/scrapy/pypydispatcher
+  license: BSD-3-Clause-Clear
+  license_family: BSD
+  license_file: license.txt
+  summary: 'Multi-producer-multi-consumer signal dispatching mechanism, with PyPy support'
+
+  description: |
+    Multi-producer-multi-consumer signal dispatching mechanism, with PyPy support
+
+extra:
+  recipe-maintainers:
```

Comment:
@dangra @rmax Are you happy to be listed as recipe maintainers here?

Reply:
@chrisburr I’ve just removed Daniel from the list of recipe maintainers.

@rmax Was https://github.com/conda-forge/staged-recipes/pull/14064#issuecomment-785698563 an OK to be a maintainer? :slightly_smiling_face:

Reply:
Yes. I'm OK to be listed as maintainer.